stubs/fpdf2/METADATA.toml

```
@@ -1,5 +1,5 @@
-version = "2.6.0"
-requires = ["types-Pillow"]
+version = "2.6.1"
+requires = ["cryptography", "types-Pillow"]
```

Doesn't look like the `cryptography` dependency is being used anywhere

```suggestion
requires = ["types-Pillow"]
```

---

Moved to correct file.